distributions/aws/apps/jupyter-web-app/configs/spawner_ui_config.yaml

```diff
@@ -0,0 +1,215 @@
+    - public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-pytorch:1.9.0-cpu-py38
+  imageGroupOne:
+    # The container Image for the user's Group One Server
+    # The annotation `notebooks.kubeflow.org/http-rewrite-uri: /`
+    # is applied to notebook in this group, configuring
+    # the Istio rewrite for containers that host their web UI at `/`
+    value: public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/codeserver-python:v1.4
+    # The list of available standard container Images
+    options:
+    - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/codeserver-python:v1.4
+  imageGroupTwo:
+    # The container Image for the user's Group Two Server
+    # The annotation `notebooks.kubeflow.org/http-rewrite-uri: /`
+    # is applied to notebook in this group, configuring
+    # the Istio rewrite for containers that host their web UI at `/`
+    # The annotation `notebooks.kubeflow.org/http-headers-request-set`
+    # is applied to notebook in this group, configuring Istio
+    # to add the `X-RStudio-Root-Path` header to requests
+    value: public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/rstudio-tidyverse:v1.4
+    # The list of available standard container Images
+    options:
+    - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/rstudio-tidyverse:v1.4
+  # If true, hide registry and/or tag name in the image selection dropdown
+  hideRegistry: true
+  hideTag: false
+  allowCustomImage: true
+  # If true, users can input custom images
+  # If false, users can only select from the images in this config
+  imagePullPolicy:
+    # Supported values: Always, IfNotPresent, Never
+    value: IfNotPresent
+    readOnly: false
+  cpu:
+    # CPU for user's Notebook
+    value: '0.5'
+    # Factor by with to multiply request to calculate limit
+    # if no limit is set, to disable set "none"
+    limitFactor: "1.2"
+    readOnly: false
+  memory:
+    # Memory for user's Notebook
+    value: 1.0Gi
+    # Factor by with to multiply request to calculate limit
+    # if no limit is set, to disable set "none"
+    limitFactor: "1.2"
+    readOnly: false
+  environment:
+    value: {}
+    readOnly: false
+  workspaceVolume:
+    # Workspace Volume to be attached to user's Notebook
+    # Each Workspace Volume is declared with the following attributes:
+    # Type, Name, Size, MountPath and Access Mode
+    value:
+      type:
+        # The Type of the Workspace Volume
+        # Supported values: 'New', 'Existing'
+        value: New
+      name:
+        # The Name of the Workspace Volume
+        # Note that this is a templated value. Special values:
+        # {notebook-name}: Replaced with the name of the Notebook. The frontend
+        #                  will replace this value as the user types the name
+        value: 'workspace-{notebook-name}'
+      size:
+        # The Size of the Workspace Volume (in Gi)
+        value: '5Gi'
+      mountPath:
+        # The Path that the Workspace Volume will be mounted
+        value: /home/jovyan
```

Comment:
Whats this?

Reply:
path where PV created for the notebook server is mounted to the server pod.

the only change in this file is the notebook-servers ecr images. Rest is same as [upstream](https://github.com/awslabs/kubeflow-manifests/blob/v1.4-branch/apps/jupyter/jupyter-web-app/upstream/base/configs/spawner_ui_config.yaml)